### DISCUSSION TITLE
iPhone does not send INVITE when making a call

### DISCUSSION THREAD

**Discussion by ktsubouc** [herlesupreeth/docker_open5gs#337] [Q&A]
Dear all,

I am testing 4G VoLTE call by docker_open5gs. 
I confirmed that Android phone can make calls with each other. 
However iPhone (I tested iPhone SE3, iPhone 12, iPhone 13, iPhone 15) cannot make calls. Specifically speaking, iPhone registers/subscribes to IMS, but it does not send INVITE message when it makes a call. On a side note, I observed iPhone replied with "486 Establising Another Call" to OPTIONS message while iPhone was making a call.
Interestingly, iPhone can receive calls from Android.
Any idea to solve this issue?

Many thanks in advance!
Koji

**Comment by herlesupreeth** [comment]
Try changing `tcp_connection_lifetime` to 1 in kamailio_pcscf.cfg file under pcscf folder and restart P-CSCF

**Reply by ktsubouc** [reply]
thanks a lot for the suggestion. I have done that, but the behavior has not changed.

**Reply by herlesupreeth** [reply]
Then, please attach a pcap of scenario where you register the iPhone to network and followed by it try to place a call.

**Reply by ktsubouc** [reply]
[docker_open5gs_call_log_sip_only_2024-07-17.pcapng.zip](https://github.com/user-attachments/files/16264701/docker_open5gs_call_log_sip_only_2024-07-17.pcapng.zip)

pcap is attached. this was taken at S1 interface.

id:8111000100101 - iPhone12
id:8111000100102 - Android phone

**Reply by ktsubouc** [reply]
You will see "486 Establishing Another Call" e.g. frame#46 when I was making a call from iPhone.

**Reply by herlesupreeth** [reply]
Thanks for the pcap. But sadly I couldnt narrow down why there is 486  Establishing Another Call. One thing you can try is to disable NAT ping (using SIP OPTIONS)

change `#!define WITH_NATPING` to `##!define WITH_NATPING` in pcscf.cfg

**Reply by ktsubouc** [reply]
Thanks for replying and sorry for not responding for a while. I have just tested with the latest build by disabling NAT ping, but it did not work.

**Comment by ajrise** [comment]
This is a call I made through iPhone XSMax, it's send the Invite normally
![image](https://github.com/user-attachments/assets/46920dc1-c160-469c-9b5d-57eb413fd661)

**Reply by ktsubouc** [reply]
Thanks, which version are you using? I am using the latest one. The pcap taken was done by the version downloaded today, but I have been struggling with this issue for a couple of weeks.

**Reply by ajrise** [reply]
It's IOS17.2.1
